common/util.h

```diff
@@ -109,6 +110,7 @@ class ExitHandler {
   ExitHandler() {
     std::signal(SIGINT, (sighandler_t)set_do_exit);
     std::signal(SIGTERM, (sighandler_t)set_do_exit);
+    std::signal(SIGHUP, (sighandler_t)set_do_exit);
```

Comment:
```suggestion
```

Reply:
this was from tmux exiting most likely, and we only saw it first because msgq was stealing our signal handler (fixed in https://github.com/commaai/openpilot/pull/34410)